packages/backend/src/clients/github/Github.ts

```diff
@@ -158,23 +176,28 @@ export const createBranch = async ({
     repo,
     sha,
     branch,
-    installationId,
+    token,
 }: {
     owner: string;
     repo: string;
     sha: string;
     branch: string;
-    installationId: string;
+    token: string;
 }) => {
-    const octokit = getOctokitRestForApp(installationId);
+    const { octokit, headers } = getOctokitRestForUser(token);
```

Comment:
in most functions, we can use the `user` octokit, and it makes no difference. (The token still comes from the installationId if available )